miden-tx/src/executor/mod.rs

```diff
@@ -139,9 +139,9 @@ impl<D: DataStore> TransactionExecutor<D> {
         account_id: AccountId,
         block_ref: u32,
         notes: &[NoteId],
-        tx_script: Option<TransactionScript>,
+        tx_args: TransactionArgs,
```

Comment:
I wonder if we should change `tx_args` parameter type to `Option<TransactionArgs>`. Then, internally we could use `.unwrap_or_default()`.

Reply:
Jup, that would be more consistent